src/services/players/players.service.ts

```diff
@@ -57,16 +62,40 @@ class PlayersService {
   };
 
   updatePlayerVehicle = (id: string, vehicleId: string, update: PlayerVehicleUpdate) => {
-    // check player exists
-    // check vehicle exists and belongs to player
-    // update vehicle
-    // return updated vehicle
+    const player = this.getPlayer(id);
+    let playerVehicle: PlayerVehicle = vehicles.find(
```

Comment:
on small nit: this can technically be undefined
```suggestion
    let playerVehicle: undefined | PlayerVehicle = vehicles.find(
```